recipes/LibriSpeech/ASR/transformer/hparams/branchformer_large.yaml

```diff
@@ -0,0 +1,291 @@
+# ############################################################################
+# Model: E2E ASR with Transformer
+# Encoder: Branchformer Encoder
+# Decoder: Transformer Decoder + (CTC/ATT joint) beamsearch + TransformerLM
+# Tokens: unigram
+# losses: CTC + KLdiv (Label Smoothing loss)
+# Training: Librispeech 960h
+# Authors:  Titouan Parcollet
+# ############################################################################
+# Seed needs to be set at top of yaml, before objects with parameters are made
+
+seed: 7775
+__set_seed: !apply:torch.manual_seed [!ref <seed>]
+output_folder: !ref results/conformer_small/<seed>
```

Comment:
rename it to branchformer_large

Reply:
@Adel-Moumen I agree with the latest comment of @Gastron could you do it and merge please?